### PR TITLE
User lambda + Tests

### PR DESCRIPTION
**Pull request by christophercolumbusdog**
Added the user lambda code, along with relevant tests, to the UserLambdas package

### CODE REVIEW

File: lambdas/testFunction/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/cygnus/Documents/cs428/chartr/lambdas/testFunction/src/main/java/chartr/TestFunction.java
```
Comment (gmvalentino8) [inline lambdas/testFunction/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst]: Should we be Git ignoring this?

File: lambdas/testFunction/target/maven-archiver/pom.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
+#Mon Feb 12 20:02:44 CST 2018
+version=1.0-SNAPSHOT
+groupId=edu.illinois.chartr
+artifactId=testFunction
```
Comment (gmvalentino8) [inline lambdas/testFunction/target/maven-archiver/pom.properties]: Do we still need the testFunction?

File: lambdas/testFunction/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst (hunk 1 above)
Comment (christophercolumbusdog) [reply lambdas/testFunction/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst]: It's still hooked up to an endpoint on Chartr and serves as a good reference for how to write the lambda code (in our environment) in Java. I would prefer leaving it in the codebase for now.

Comment (gmvalentino8) [reply lambdas/testFunction/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst]: OK! Everything else looks good to me then

**Comment by christophercolumbusdog** [comment]
Closing up PR, integrating with Dev

**Pull request merged**